<?xml version="1.0"?>
<launch>
  <arg name="x" default="0.0" />
  <arg name="y" default="0.0" />
  <arg name="z" default="0.0" />
  <arg name="roll" default="0.0" />
  <arg name="pitch" default="0.0" />
  <arg name="yaw" default="0.0" />
  <arg name="sdf_robot_file" default="$(find up_the_stairs)/models/bd_droid/model.sdf"/>
  <arg name="robot_name" default="bd_droid"/>
  
  <node name="$(arg robot_name)_spawn_urdf" pkg="gazebo_ros" type="spawn_model" respawn="false" output="screen" args="-file $(arg sdf_robot_file) -sdf -x $(arg x) -y $(arg y) -z $(arg z) -R $(arg roll) -P $(arg pitch) -Y $(arg yaw) -model $(arg robot_name)"/>

  <node name="$(arg robot_name)_tf2_broadcaster" pkg="up_the_stairs" type="robot_tf2_broadcast.py" respawn="false" output="screen" >
      <param name="robot_name" type="string" value="bd_droid" />
  </node>
</launch>